app.js

```diff
@@ -50,7 +64,19 @@ console.log(
 //    that checks if the year is a leap year, and returns the correct
 //    number of days for Feb. The default value for the year should be
 //    the current year
-function daysInMonth() {}
+let daysPerMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+let daysPerMonthLeapYear = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+function daysInMonth(month, year = '2022') {
+    if (month >= 1 && month <= 12) {
+        if (year % 4 === 0) {
+            return daysPerMonthLeapYear[month - 1];
+        } else {
+            return daysPerMonth[month - 1];
+        }
+    } else {
+        console.log('Value is not between 1 and 12. Please enter a value between 1 and 12');
+    }
+}
```

Comment:
the first console log is returning false. you just need to tweak your function to align with the parameters set above to still return true.